node/src/accountant/mod.rs

```diff
@@ -91,7 +93,7 @@ pub struct Accountant {
     scan_schedulers: ScanSchedulers,
     financial_statistics: Rc<RefCell<FinancialStatistics>>,
     outbound_payments_instructions_sub_opt: Option<Recipient<OutboundPaymentsInstructions>>,
-    blockchain_bridge_pps_sub_opt: Option<Recipient<PayablePaymentsSetup>>,
+    payable_payments_setup_msg_payload_sub_opt: Option<Recipient<PayablePaymentsSetupMsgPayload>>,
```

Comment:
Okay, let's talk about names.

The first message from the Accountant to the BlockchainBridge says, "We need to pay these bills. Give us an agent that knows things about the blockchain we'll use to pay them, and send us back a copy of the list of bills so we don't have to remember it." - QualifiedPayablesMessage? Something like that?

The second message from the BlockchainBridge to the Accountant says, "Here's an agent that knows about the blockchain. Ask it some questions and get back to me." It also will travel to the Neighborhood, where it asks, "Which of these bills are most important to pay?" BlockchainAgentWithContextMessage?

There must be a third message (OutboundPaymentsInstructions) from somewhere to the BlockchainBridge saying, "Okay, here are the final instructions: pay the following amounts to the following Nodes."